dulwich/refs.py

```diff
@@ -882,3 +882,9 @@ def write_info_refs(refs, store):
 
 def is_local_branch(x):
     return x.startswith(b'refs/heads/')
+
+
+def strip_peeled_refs(refs):
+    """Remove all peeled refs"""
+    return {ref: sha for (ref, sha) in refs.items()
+            if not ref.endswith(b"^{}")}
```

Comment:
Please use ANNOTATED_TAG_SUFFIX rather than b"^{}"

Reply:
Didn't realize this existed, changing...